### PR TITLE
[keycloak] Allow to choose restartPolicy

### PR DESCRIPTION
**Pull request by fmasion**
Here a PR for the issue #54 i made previously

Thanks in advance

Fred

### CODE REVIEW

File: charts/keycloak/README.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ Parameter | Description | Default
 `keycloak.podAnnotations` | Extra annotations to add to pod | `{}`
 `keycloak.hostAliases` | Mapping between IP and hostnames that will be injected as entries in the pod's hosts files | `[]`
 `keycloak.enableServiceLinks` | Indicates whether information about services should be injected into pod's environment variables, matching the syntax of Docker links | `false`
+`keycloak.restartPolicy` | Pod restart Policy. One of `Always`,`OnFailure`or `Never` | `Always`
```
Comment (unguiculus) [inline charts/keycloak/README.md]: Just a little nit:
```suggestion
`keycloak.restartPolicy` | Pod restart Policy. One of `Always`, `OnFailure`, or `Never` | `Always`
```

**Comment by unguiculus** [comment]
You will have to sign off your commits. See https://github.com/codecentric/helm-charts/pull/55/checks?check_run_id=152678009

**Comment by fmasion** [comment]
Hi, 
I think signoff is ok. 
Thank you

**Comment by fmasion** [comment]
Vielen Dank Reinhard

**Pull request merged**